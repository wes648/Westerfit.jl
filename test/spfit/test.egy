     3    1          0.000000          0.000000   1.000000    2:  0  0  0  1
     3    2          0.111060          0.000000   1.000000    2:  1  1  0  1
     4    1          0.104925          0.000000   1.000000    2:  1  1  1  1
     4    2          0.088796          0.000000   1.000000    2:  1  0  1  1
     5    1          0.205438          0.000000   0.882149    4:  1  1  0  2
     5    2          0.283773          0.000000   1.000000    4:  2  0  2  2
     5    3          0.470154          0.000000   1.000000    4:  2  2  0  2
     5    4          0.326369          0.000000   0.882149    4:  2  1  2  2
     6    1          0.177649          0.000000   0.920526    4:  1  1  1  2
     6    2          0.108580          0.000000   0.998870    4:  1  0  1  2
     6    3          0.366428          0.000000   0.920526    4:  2  1  1  2
     6    4          0.467755          0.000000   0.998870    4:  2  2  1  2
     7    1          0.313582          0.000000   0.996746    6:  2  0  2  3
     7    2          0.644384          0.000000   0.820525    6:  2  2  0  3
     7    3          0.360585          0.000000   0.967038    6:  2  1  2  3
     7    4          0.838617          0.000000   0.817271    6:  3  2  2  3
     7    5          0.695438          0.000000   0.967176    6:  3  1  2  3
     7    6          1.073740          0.000000   0.999862    6:  3  3  0  3
     8    1          0.435514          0.000000   0.939030    6:  2  1  1  3
     8    2          0.642318          0.000000   0.819590    6:  2  2  1  3
     8    3          0.602475          0.000000   0.939300    6:  3  1  3  3
     8    4          1.073369          0.000000   0.999730    6:  3  3  1  3
     8    5          0.569489          0.000000   0.979798    6:  3  0  3  3
     8    6          0.851165          0.000000   0.839792    6:  3  2  1  3
     9    1          0.965271          0.000000   0.525603    8:  3  2  2  4
     9    2          0.776045          0.000000   0.954269    8:  3  1  2  4
     9    3          1.338255          0.000000   0.763700    8:  3  3  0  4
     9    4          0.923939          0.000000   0.547969    8:  4  0  4  4
     9    5          1.288942          0.000000   0.926544    8:  4  2  2  4
     9    6          1.922869          0.000000   0.999885    8:  4  4  0  4
     9    7          0.964828          0.000000   0.955050    8:  4  1  4  4
     9    8          1.577651          0.000000   0.762918    8:  4  3  2  4
    10    1          0.635119          0.000000   0.981220    8:  3  1  3  4
    10    2          1.338359          0.000000   0.767123    8:  3  3  1  4
    10    3          0.607203          0.000000   0.994312    8:  3  0  3  4
    10    4          0.967142          0.000000   0.907212    8:  3  2  1  4
    10    5          1.127940          0.000000   0.980503    8:  4  1  3  4
    10    6          1.579902          0.000000   0.767839    8:  4  3  1  4
    10    7          1.248666          0.000000   0.901652    8:  4  2  3  4
    10    8          1.922825          0.000000   0.999872    8:  4  4  1  4
    11    1          0.983011          0.000000   0.992686   10:  4  0  4  5
    11    2          1.412804          0.000000   0.941310   10:  4  2  2  5
    11    3          2.280996          0.000000   0.716562   10:  4  4  0  5
    11    4          0.997544          0.000000   0.987064   10:  4  1  4  5
    11    5          1.750717          0.000000   0.845933   10:  4  3  2  5
    11    6          1.751070          0.000000   0.934284   10:  5  2  4  5
    11    7          2.555035          0.000000   0.716273   10:  5  4  2  5
    11    8          1.656908          0.000000   0.959554   10:  5  1  4  5
    11    9          2.116250          0.000000   0.873529   10:  5  3  2  5
    11   10          3.016797          0.000000   0.999914   10:  5  5  0  5
    12    1          1.220682          0.000000   0.955484   10:  4  1  3  5
    12    2          1.751053          0.000000   0.866041   10:  4  3  1  5
    12    3          1.338533          0.000000   0.780766   10:  4  2  3  5
    12    4          2.281053          0.000000   0.717144   10:  4  4  1  5
    12    5          1.412920          0.000000   0.957197   10:  5  1  5  5
    12    6          2.106095          0.000000   0.864415   10:  5  3  3  5
    12    7          3.016792          0.000000   0.999913   10:  5  5  1  5
    12    8          1.412419          0.000000   0.821080   10:  5  0  5  5
    12    9          1.839551          0.000000   0.959848   10:  5  2  3  5
    12   10          2.555339          0.000000   0.716981   10:  5  4  1  5
    13    1          1.843083          0.000000   0.850103   12:  5  2  4  6
    13    2          2.790213          0.000000   0.828549   12:  5  4  2  6
    13    3          1.761561          0.000000   0.947787   12:  5  1  4  6
    13    4          2.283981          0.000000   0.912853   12:  5  3  2  6
    13    5          3.470122          0.000000   0.677867   12:  5  5  0  6
    13    6          1.947488          0.000000   0.875362   12:  6  0  6  6
    13    7          2.502399          0.000000   0.973981   12:  6  2  4  6
    13    8          3.195882          0.000000   0.829375   12:  6  4  2  6
    13    9          4.355384          0.000000   0.999933   12:  6  6  0  6
    13   10          1.945788          0.000000   0.951164   12:  6  1  6  6
    13   11          2.731805          0.000000   0.909661   12:  6  3  4  6
    13   12          3.774830          0.000000   0.677683   12:  6  5  2  6
    14    1          1.445509          0.000000   0.989916   12:  5  1  5  6
    14    2          2.246894          0.000000   0.571916   12:  5  3  3  6
    14    3          3.470133          0.000000   0.677940   12:  5  5  1  6
    14    4          1.438647          0.000000   0.992187   12:  5  0  5  6
    14    5          1.977606          0.000000   0.956485   12:  5  2  3  6
    14    6          2.790244          0.000000   0.827891   12:  5  4  1  6
    14    7          2.305345          0.000000   0.638179   12:  6  1  5  6
    14    8          2.762148          0.000000   0.923826   12:  6  3  3  6
    14    9          3.774866          0.000000   0.677767   12:  6  5  1  6
    14   10          2.345675          0.000000   0.949110   12:  6  2  5  6
    14   11          3.194115          0.000000   0.827520   12:  6  4  3  6
    14   12          4.355383          0.000000   0.999933   12:  6  6  1  6
