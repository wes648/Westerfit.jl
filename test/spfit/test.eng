   0,   1,  -1,   1,   0,   0,    -0.1225592745
   0,   1,   0,   1,   0,   0,    -0.1225592745
   0,   1,   1,   0,   0,   0,    -0.0000000000
   2,   0,   0,   0,   0,   0,     0.0000000000
   2,   1,  -1,   1,   0,   0,    -0.1225592745
   2,   1,   0,   1,   0,   0,    -0.1225592745
   2,   1,   1,   0,   0,   0,    -0.0000000000
   2,   2,  -2,   1,   0,   0,     0.0000000000
   2,   2,  -1,   1,   0,   0,     0.0000000000
   2,   2,   0,   2,   0,   0,    -0.2451185489
   2,   2,   1,   2,   0,   0,    -0.2451185489
   2,   2,   2,   0,   0,   0,     0.0000000000
   4,   1,  -1,   1,   0,   0,     0.1225592745
   4,   1,   0,   1,   0,   0,     0.0000000000
   4,   1,   1,   0,   0,   0,     0.1225592745
   4,   2,  -2,   1,   0,   0,     0.0000000000
   4,   2,  -1,   1,   0,   0,     0.0000000000
   4,   2,   0,   2,   0,   0,    -0.1225592745
   4,   2,   1,   2,   0,   0,    -0.1225592745
   4,   2,   2,   0,   0,   0,     0.0000000000
   4,   3,  -3,   1,   0,   0,     0.0000000000
   4,   3,  -2,   1,   0,   0,     0.0000000000
   4,   3,  -1,   3,   0,   0,    -0.3676778234
   4,   3,   0,   3,   0,   0,    -0.3676778234
   4,   3,   1,   2,   0,   0,    -0.2451185489
   4,   3,   2,   2,   0,   0,    -0.2451185489
   4,   3,   3,   0,   0,   0,     0.0000000000
   6,   2,  -2,   1,   0,   0,     0.2451185489
   6,   2,  -1,   1,   0,   0,     0.1225592745
   6,   2,   0,   2,   0,   0,    -0.0000000000
   6,   2,   1,   2,   0,   0,     0.1225592745
   6,   2,   2,   0,   0,   0,     0.2451185489
   6,   3,  -3,   1,   0,   0,     0.0000000000
   6,   3,  -2,   1,   0,   0,    -0.0000000000
   6,   3,  -1,   3,   0,   0,    -0.2451185489
   6,   3,   0,   3,   0,   0,    -0.2451185489
   6,   3,   1,   2,   0,   0,    -0.1225592745
   6,   3,   2,   2,   0,   0,    -0.1225592745
   6,   3,   3,   0,   0,   0,     0.0000000000
   6,   4,  -4,   1,   0,   0,     0.0000000000
   6,   4,  -3,   1,   0,   0,    -0.0000000000
   6,   4,  -2,   3,   0,   0,    -0.3676778234
   6,   4,  -1,   3,   0,   0,    -0.3676778234
   6,   4,   0,   4,   0,   0,    -0.4902370978
   6,   4,   1,   4,   0,   0,    -0.4902370978
   6,   4,   2,   2,   0,   0,    -0.0000000000
   6,   4,   3,   2,   0,   0,    -0.0000000000
   6,   4,   4,   0,   0,   0,     0.0000000000
   8,   3,  -3,   1,   0,   0,     0.3676778234
   8,   3,  -2,   1,   0,   0,     0.2451185489
   8,   3,  -1,   3,   0,   0,     0.1225592745
   8,   3,   0,   3,   0,   0,    -0.0000000000
   8,   3,   1,   2,   0,   0,     0.1225592745
   8,   3,   2,   2,   0,   0,     0.2451185489
   8,   3,   3,   0,   0,   0,     0.3676778234
   8,   4,  -4,   1,   0,   0,     0.0000000000
   8,   4,  -3,   1,   0,   0,     0.0000000000
   8,   4,  -2,   3,   0,   0,    -0.1225592745
   8,   4,  -1,   3,   0,   0,    -0.1225592745
   8,   4,   0,   4,   0,   0,    -0.2451185489
   8,   4,   1,   4,   0,   0,    -0.2451185489
   8,   4,   2,   2,   0,   0,     0.0000000000
   8,   4,   3,   2,   0,   0,     0.0000000000
   8,   4,   4,   0,   0,   0,     0.0000000000
   8,   5,  -5,   1,   0,   0,     0.0000000000
   8,   5,  -4,   1,   0,   0,     0.0000000000
   8,   5,  -3,   3,   0,   0,    -0.3676778234
   8,   5,  -2,   3,   0,   0,    -0.3676778234
   8,   5,  -1,   5,   0,   0,    -0.6127963723
   8,   5,   0,   5,   0,   0,    -0.6127963723
   8,   5,   1,   4,   0,   0,    -0.4902370978
   8,   5,   2,   4,   0,   0,    -0.4902370978
   8,   5,   3,   2,   0,   0,    -0.0000000000
   8,   5,   4,   2,   0,   0,    -0.0000000000
   8,   5,   5,   0,   0,   0,     0.0000000000
