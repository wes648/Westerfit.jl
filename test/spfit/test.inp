test
%CNTRLS
NFOLD = 0  
S = 0.5
TK = 25.0
assign = eeo
RUNmode = F
mcalc = 0
vtmax = 0
Jmax = 4.5
νmin =   0.0
νmax = 100.0

%2NDORDER
A;                3666.961;   1.0
B;                1833.626;   1.0
C;                1231.939;   1.0
Dab;                   0.0;   0.0
ϵzz;              3520.882;   1.0
ϵxx;              821.2990;   1.0
ϵyy;              -14.2460;   1.0
ϵxz;                   0.0;   0.0
χzz;                   0.0;   0.0
χxmy;                  0.0;   0.0
χxz;                   0.0;   0.0
F;                     0.0;   0.0
ρ;                     0.0;   0.0
Vn;                    0.0;   0.0
η;                     0.0;   0.0

%PARAMS N^a Nz^b (N₊^c + N₋^c) (NS)^d Sz^e Pₐ^f cos(g*α) sin(h*α) Ny^(1-δ(0,h))
%Op;           Val;   Scl;   a;   b;   c;   d;   e;   f;   g;   h;  stg
μa;            1.0;   0.0;   0;   1;   0;   0;   0;   0;   0;   0;   0
μb;            1.0;   0.0;   0;   0;   1;   0;   0;   0;   0;   0;   0
ΔN;            0.0;   0.0;   4;   0;   0;   0;   0;   0;   0;   0;   1
ΔNK;           0.0;   0.0;   2;   2;   0;   0;   0;   0;   0;   0;   1
ΔK;            0.0;   0.0;   0;   4;   0;   0;   0;   0;   0;   0;   1
δN;            0.0;   0.0;   2;   0;   2;   0;   0;   0;   0;   0;   1
δK;            0.0;   0.0;   0;   2;   2;   0;   0;   0;   0;   0;   1


