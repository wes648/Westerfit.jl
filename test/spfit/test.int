test
1101  8  3048.1982   0  14   -9.0  -6.0    18.00  8.0
   001   2.331
   002   2.331
