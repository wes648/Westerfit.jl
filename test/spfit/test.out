test   @   2023-06-09T22:00:14.758

Control Parameters
NFOLD = 0  
S = 0.5
TK = 25.0
assign = eeo
RUNmode = F
mcalc = 0
vtmax = 0
Jmax = 4.5
νmin =   0.0
νmax = 100.0

Initial Parameters
BN;                    2134.1785;   1.0
BK;                    1532.7825;   1.0
B⨦;           150.42174999999997;   1.0
Dab;                         0.0;   0.0
T⁰₀(ϵ);      -2498.7344372852035;   1.0
T²₀(ϵ);       2545.3101072862423;   1.0
T²₁(ϵ);                     -0.0;   0.0
T²₂(ϵ);                 417.7725;   1.0
T²₀(χ);                      0.0;   0.0
T²₁(χ);                     -0.0;   0.0
T²₂(χ);                      0.0;   0.0
F;                           0.0;   0.0
-2ρF;                       -0.0;   0.0
V3/2;                        0.0;   0.0
η;                           0.0;   0.0

μa;                          0.0;   0.0
μb;                          0.0;   0.0
ΔN;                          0.0;   0.0
ΔNK;                         0.0;   0.0
ΔK;                          0.0;   0.0

Number of lines = 453

Initial RMS = 2.811381183452301e-5 MHz, Initial λ = 2.811381143932981e-12

-------------------------------------

            2134.178500548159;     0.0000
BK;           1532.7824998389226;    -0.0000
B⨦;           150.42175004250436;     0.0000
Dab;                         0.0;      Fixed
T⁰₀(ϵ);       -2498.734438083115;    -0.0000
T²₀(ϵ);       2545.3101075622913;     0.0000
T²₁(ϵ);                     -0.0;      Fixed
T²₂(ϵ);       417.77249847069396;    -0.0000
T²₀(χ);                      0.0;      Fixed
T²₁(χ);                     -0.0;      Fixed
T²₂(χ);                      0.0;      Fixed
F;                           0.0;      Fixed
-2ρF;                       -0.0;      Fixed
V3/2;                        0.0;      Fixed
η;                           0.0;      Fixed
μa;                          0.0;      Fixed
μb;                          0.0;      Fixed
ΔN;                          0.0;      Fixed
ΔNK;                         0.0;      Fixed
ΔK;                          0.0;      Fixed

Max Change; T²₂(ϵ), 0.0000%

-------------------------------------

